hw2/alan0201tw/Q2/makefile

```diff
@@ -0,0 +1,14 @@
+
+main: vector.cpython-36m-x86_64-linux-gnu.so
+
+vector.cpython-36m-x86_64-linux-gnu.so : vector.cpp
+	g++ -O3 -Wall -shared -std=c++17 -fPIC `python3 -m pybind11 --includes` vector.cpp -o vector`python3-config --extension-suffix`
+
+clean:
+	@echo "Cleaning...";
+	@echo "$(RM) *.so"; $(RM) *.so
+
+test:
+	nosetests -v -s pybind_test.py

```

Comment:
@alan0201tw  I cannot run `make test`
Please revise it to make it run on `nsd-ami`

Reply:
Sorry for your troubles.
But I believe this is due to the fact that 'nose' is not installed by default on nsd-ami.
My previous testing requires me to manually install it by 'pip install nose'.

Please inform me about whether I should update the dependencies or do anything, since I believe it's not reasonable for me to just stick an 'install' command in my makefile.

Again, sorry for your trouble.